elasticsearch/client_test.go

```diff
@@ -265,6 +270,58 @@ func TestAddDocument(t *testing.T) {
 	})
 }
 
+func TestBulkUpdate(t *testing.T) {
+	testSetup(t)
+	esIndex := "search_index"
+	esDestType := "docType"
+	esDestIndex := fmt.Sprintf("%s/%s", esIndex, esDestType)
```

Comment:
Why do we need the esDestType? Remember this is working with Elasticsearch version 7, so the path does not handle types.

Reply:
This again would be confusing for a developer working in this library. I would suggest collapsing lines 275 to 277 into:

```
        esDestIndex := "ons_test"
```